Examples/Swift/ExtrusionsExample.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ExtrusionsExample.swift
+//  Examples
+//
+//  Created by Jordan Kiley on 5/10/17.
+//  Copyright © 2017 Mapbox. All rights reserved.
+//
+
+import Mapbox
+
+@objc(ExtrusionsExample_Swift)
+
+class ExtrusionsExample: UIViewController, MGLMapViewDelegate {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let mapView = MGLMapView(frame: view.bounds, styleURL: MGLStyle.lightStyleURL(withVersion: 9))
+        
+        // Center the map on the Colosseum in Rome, Italy.
+        mapView.setCenter(CLLocationCoordinate2D(latitude: 41.8902, longitude: 12.4922), animated: false)
+
+        // Set the map view camera's pitch and distance.
+        mapView.camera = MGLMapCamera(lookingAtCenter: mapView.centerCoordinate, fromDistance: 600, pitch: 60, heading: 0)
+        mapView.delegate = self
+        
+        view.addSubview(mapView)
+    }
+    
+    func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle) {
+        
+        // Access the Mapbox Streets source and use it to create a `MGLFillExtrusionLayer`.
+        let source = style.source(withIdentifier: "composite")
```

Comment:
For a more robust solution (which may be overkill for this particular example), see [this extension method](https://github.com/mapbox/mapbox-navigation-ios/blob/20c082eea7946bb35b5895f8ca525275ce0bdc7f/MapboxNavigation/MGLVectorSource.swift#L5-L10) used in the navigation SDK. You’d filter `style.sources` by `isMapboxStreets` and [create a Streets source](https://github.com/mapbox/mapbox-navigation-ios/blob/238e3f2a52decba5489a0b44872bb6168fd59115/MapboxNavigation/RouteMapViewController.swift#L332-L343) if there isn’t one.